metadata.rb

```diff
@@ -168,3 +176,11 @@
 attribute "kube-hops/kfserving/img_tar_url",
           :description =>  "Remote container images registry from which to fetch the kfserving and dependencies images",
           :type => 'string'
+
+attribute "kube-hops/serving_node_labels",
+          :description =>  "The labels to apply to the KfServing pods",
```

Comment:
```suggestion
          :description =>  "The labels used for node selection in model serving pods, in the format key1=value1,key2=value2",
```

We also apply node selection when serving models with regular kubernetes deployments